Feed/views.py

```diff
@@ -0,0 +1,5 @@
+from django.shortcuts import render
+
+
+def Feed(request):
```

Comment:
The method name should start with a lower case letter, I know you base on my PR and I changed it so maybe edit those changes out of the PR or fix it according to the name standards of methods like Nati mentioned in my PR.

Reply:
Yes I'm waiting for your new changes to rebase my PR according to yours